beamdriver.py

```diff
@@ -21,7 +21,9 @@ class SimServer(SimpleServer):
         'HOPR': 'display.limitHigh',
         'LOPR': 'display.limitLow',
         'DESC': 'display.description',
-        'EGU': 'display.units'
+        'EGU': 'display.units',
+        'DRVL': 'control.limitLow',
+        'DRVH': 'control.limitHigh',
```

Comment:
nit: it annoys me a bit these are flipped relative to the other changes

Reply:
fixed